### ISSUE TITLE
  "SnipePokemonNotInPokedex": false, Still trying to snipe legendaries

### ISSUE THREAD

**Issue by elmaarmut** [NECROBOTIO/NecroBot#3368]
Title is self explanatory. I have:

  "SnipePokemonNotInPokedex": false, 

Yet I am entering an andless loop of:

![1](https://cloud.githubusercontent.com/assets/20822856/17495152/8f13b42a-5dc0-11e6-8375-24b08dc542bb.jpg)

**Comment by nfettinger** [comment]
Try adding the legendaries to the ignore list and setting the config item for that list on.

**Comment by JaEdmuva** [comment]
Hey there... that was spam in the sniping servers, we all saw that.

**Comment by tangdami** [comment]
yep, somebody posted those nonsense on sniper channel, just ignore that.

**Comment by ChemiHatake** [comment]
Check the   "PokemonToSnipe" filter

**Comment by elmaarmut** [comment]
My point was that since the legendaries are not in my pokedex, and "SnipePokemonNotInPokedex" is set to "false", the bot shouldn't be going after these legendaries even if they are in my list of Pokemon to snipe. Kinda beats the point of having a "SnipePokemonNotInPokedex". Anyway, nfettinger seems to have gotten my point :)

**Comment by PedroSeda** [comment]
Not really, that is a good feature so you don't have to add every Pokemon not in the pokedex to the snipe list.
